package.json

```diff
@@ -26,6 +26,7 @@
     "@types/pluralize": "0.0.29",
     "babel-eslint": "^10.0.3",
     "chai": "^4.2.0",
+    "chokidar": "^3.3.0",
```

Comment:
Interested to hear why you chose this one over the other ones?

Reply:
Nodemon: API wasn't nice
Watch: too raw
Chokidar: nice API, have used previously, claims to be a wrapper that fixes node's fs events
(Will write a detailed one when I am on laptop)